build/Packaging/AzureSiteExtension/Content/install.ps1

```diff
@@ -320,7 +320,7 @@ try
 
 	if ($env:NEWRELIC_LICENSEKEY -eq $null -and $env:NEW_RELIC_LICENSE_KEY -eq $null)
```

Comment:
I left the check for both variables just in case the user set the new style at some point